### PR TITLE
[Feature] Risk calculation store

### PR DESCRIPTION
**Pull request by aleksandr-tikhonov**
Hi,

I closed previos PR https://github.com/corona-warn-app/cwa-app-ios/pull/298 to avoid of a lot of merge conflicts and open this new PR.

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift [hunk 1]
```diff
@@ -223,9 +223,20 @@ extension RiskProvider: RiskProviding {
 		}
 
 		completeOnTargetQueue(risk: risk)
+
+		saveRiskIfNeeded(risk)
```
Comment (aleksandr-tikhonov) [inline src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift:227]: @ChristianKienle could you please have a look and confirm, that this is correct place for storing previous risk level. thanks

Comment (ChristianKienle) [reply src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift:226]: yeah looking at it :)

File: src/xcode/ENA/ENA/Source/Workers/Store.swift [hunk 2]
```diff
@@ -224,6 +227,11 @@ final class SecureStore: Store {
 		set { kvStore["hourlyFetchingEnabled"] = newValue }
 	}
 
+	var previousRisk: Risk? {
```
Comment (ChristianKienle) [inline src/xcode/ENA/ENA/Source/Workers/Store.swift:230]: @llanghammer-sap I think transactions become more and more unavoidable :D Sorry. Let's talk about it once more - maybe tomorrow.

**Review by inf2381 — APPROVED** [review]

**Review by i336616 — APPROVED** [review]
Looked at how the previous summary and the riskLevelHasChanged is being handled, this seems to be ok

**Review by haosap — APPROVED** [review]

**Pull request merged**